importer/api/__init__.py

```diff
@@ -4,11 +4,13 @@
 
 from __future__ import unicode_literals
 
+from bs4 import BeautifulSoup
 from shutil import rmtree
 import logging
 from tempfile import mkdtemp
 from os.path import join, exists, isdir
 from os import listdir
+from pprint import pformat
```

Comment:
`pformat` was used during development and should be deleted. I don't know why it didn't trigger an "unused import" error. I'll take it out.